Dockerfile

```
@@ -8,15 +8,24 @@ ARG OHSOMEAPI_VERSION
 WORKDIR /opt/app/
 
 # System deps and get the ohsome-api repository data:
-RUN apk add git bash maven && git clone https://github.com/GIScience/ohsome-api.git ./ && git fetch --all --tags
+RUN apk add git bash maven curl && git clone https://github.com/GIScience/ohsome-api.git ./ && git fetch --all --tags
 
 # Checkout version if provided
 RUN if [ -z $OHSOMEAPI_VERSION ] || [ "$OHSOMEAPI_VERSION" = "latest" ] ; then echo Version not provided. Sticking to latest version.; else echo Version provided. Checkout $OHSOMEAPI_VERSION \
-    && git checkout --quiet tags/$OHSOMEAPI_VERSION ; fi
+    && git checkout --quiet tags/$OHSOMEAPI_VERSION ; fi \
+    && echo Download the fallback and test data \
+    && curl -LJO https://github.com/GIScience/ohsome-api-dockerized/raw/main/fallback_data/fallback.tar.xz \
+    && echo Extract the data. \
+    && tar -xf fallback.tar.xz \
+    && rm -rf fallback.tar.xz
```

this could be shortened slightly to:

```suggestion
    && curl -L https://github.com/GIScience/ohsome-api-dockerized/raw/main/fallback_data/fallback.tar.xz | tar -xJ
```

(which would be slightly faster and use slightly less intermediate disk space)